src/layers/image/bilinear_resize.cpp

```diff
@@ -40,6 +41,11 @@ void bilinear_resize_layer<TensorDataType, Layout, Device>::fp_compute() {
   const auto& local_input = this->get_local_prev_activations();
   auto& local_output = this->get_local_activations();
 
+  const auto& mode = this->m_model->get_execution_context().get_execution_mode();
+  if (mode != execution_mode::training) {
+    El::Copy(local_input, local_output);
+    return;
+  }
```

Comment:
The bilinear resize layer takes a fixed-size input and resizes it to a fixed-size output. I don't see how we can disable it at evaluation time.
```suggestion
```